docs/charts/Heatmap.mdx

```diff
@@ -80,6 +82,25 @@ const MyChart = () => (
 );
 ```
 
+## Color Schemes
+In addition to standard color schemes, `HeatmapSeries` also accepts a color scheme that specifies the fill, stroke, and glow for different cells.
+
+```jsx
+type HeatmapColorSchemeItem = {
```

Comment:
Is this type still valid?

Reply:
No, I pushed that last commit a little hastily and left some junk 😅